rotkehlchen/api/rest.py

```diff
@@ -2919,6 +2921,66 @@ def get_current_assets_price(
         }
         return _wrap_in_ok_result(process_result(result))
 
+    def query_asset_mappings(self, filter_query: AssetMappingsFilterQuery) -> Response:
```

Comment:
Here and in all other variables and function names change to `LocationAssetMappings`